bower.json

```diff
@@ -19,7 +19,7 @@
     "tests"
   ],
   "dependencies": {
-    "angular-route": "~1.2",
-    "djangular-rest-framework": "~1.2"
+    "angular-route": "1.2 - 1.5",
+    "djangular-rest-framework": "1.2 - 1.5"
```

Comment:
I'm not sure these ranges are correct for DRF? It's current version seems to be version 4.0?